modules/swaync/base.css

```diff
@@ -0,0 +1,182 @@
+progress,
+progressbar,
+trough {
+  border: 1px solid @base0D;
+}
+
+trough {
+  background: @base01;
+}
+
+.notification.low,
+.notification.normal {
+  border: 1px solid @base0D;
+}
+
+.notification.low progress,
+.notification.normal progress {
+  background: @base0F;
+}
+
+.notification.critical {
+  border: 1px solid @base08;
+}
+
+.notification.critical progress {
+  background: @base08;
+}
+
+.summary {
+  color: @base05;
+}
+
+.body {
+  color: @base06;
+}
+
+.time {
+  color: @base06;
+}
+
+.notification-action {
+  color: @base05;
+  background: @base01;
+  border: 1px solid @base0D;
+}
+
+.notification-action:hover {
+  background: @base01;
+  color: @base05;
+}
+
+.notification-action:active {
+  background: @base0F;
+  color: @base05;
+}
+
+.close-button {
+  color: @base02;
+  background: @base08;
+}
+
+.close-button:hover {
+  background: lighter(@base08);
+  color: lighter(@base02);
+}
+
+.close-button:active {
+  background: @base08;
+  color: @base00;
+}
+
+.notification-content {
+  background: @base00;
+  border: 1px solid @base0D;
+}
+
+.floating-notifications.background .notification-row .notification-background {
+  background: transparent;
```

Comment:
What about this?

Reply:
This needs to be left in, otherwise there will be a background outside of notifications